src/cfnlint/data/AdditionalSpecs/OnlyOne.json

```diff
@@ -38,5 +38,25 @@
     ]
   },
   "ResourceTypes": {
+    "AWS::AutoScaling::AutoScalingGroup": [
```

Comment:
Should this also be in at least one? 

`Required: Conditional. You must specify one of the following: InstanceId, LaunchConfigurationName, LaunchTemplate, or MixedInstancesPolicy.`

Reply:
Well, just specified 2 values and got the following error:

```
Exactly one of [LaunchConfigurationName,InstanceId,LaunchTemplate,MixedInstancesPolicy] needs to be specified
```